components/script/dom/htmllinkelement.rs

```diff
@@ -256,10 +255,22 @@ impl HTMLLinkElement {
 
         let referrer_policy = match self.RelList().Contains("noreferrer".into()) {
             true => Some(ReferrerPolicy::NoReferrer),
-            false => None,
+            false => document.get_referrer_policy(),
+        };
+
+        let request = RequestInit {
+            url: url.clone(),
+            type_: RequestType::Script,
```

Comment:
The spec doesn't specify a type; the table at https://fetch.spec.whatwg.org/#concept-request-type says it should be style.